apps/api/app/controllers/events_controller.rb

```diff
@@ -8,6 +10,16 @@ def index
     render json: @events
   end
 
+  # GET /events/upcoming
+  def upcoming
+    today = Time.now.beginning_of_day
+    future_date = today + 2.years
+    range = today..future_date
+    @events = Event.where(start: range)
+  
+    render json: @events, include: [:user]
```

Comment:
this looks great. the 2 year range is a bit of a code smell. let's just make it so this method returns all events that have a start after today

Reply:
I can see that

Reply:
Just updated this per your suggestions. Thanks for the help!